test/routes/form.test.js

```diff
@@ -745,7 +745,6 @@ describe('GET /supervisor/review', () => {
         expect(res.text).not.toContain('overriddenCategoryT')
         expect(res.text).not.toContain('overriddenCategoryJ')
         expect(res.text).toContain('overriddenCategoryR')
-        expect(res.text).not.toContain('value="R" checked')
```

Comment:
Why was this changed?

Reply:
I dont touch it tbh. is there something i need to update in it before I push the commented changes?

Reply:
I'm not sure why this is showing as removal. On main there is a line: `expect(res.text).not.toContain('value="R" checked')` Maybe something has gone wrong with merging the main back in? Any ideas @Olusola-Opencast ?